data/Makefile

```diff
@@ -90,6 +103,47 @@ endif
 .PHONY: f1chr4 f1chr2R f1 fishchr25 fish
 all: f1chr4 f1chr2R f1 fishchr25 fish
 
+physical-map:
+
+ifndef ref
+physical-map: \
+	$(lr).k$k-w$w.n$(n1)-$(n2).c2-$(minimizer_multiplicity).physlr.overlap.m$m.mol.mol2-bcs.backbone.path
+else
+physical-map: \
+	$(lr).k$k-w$w.n$(n1)-$(n2).c2-$(minimizer_multiplicity).physlr.overlap.m$m.mol.mol2-bcs.backbone.map.$(ref).n10.paf.gz.pdf
+endif
+# Help
+help:
+	@echo "Usage: ./make [COMMAND] [OPTION=VALUE]..."
+	@echo "    Commands:"
+	@echo ""
+	@echo "	physical-map		run physlr and generate a physical map with linked reads (10X or stLFR)"
+	@echo "	scaffold	run physlr and scaffold an existing assembly with a physical map"
+	@echo "	help            display this help page"
+	@echo "	version         display the software version"
+	@echo ""
+	@echo "    Options:"
+	@echo "	draft           draft name [draft]. File must have .fa extension"
+	@echo "	ref           ref name [draft]. File must have .fa extension. If provided, will generate Quast metrics and backbone metrics relative to reference"
+	@echo "	lr           read name [reads]. File must have .fq.gz extension"
+	@echo "	t		number of threads used [16]"
+	@echo "	minimizer_overlap		% of edges to remove in the overlap graph [90]. Available options are stLFR, 10X, or [0-100]."
+	@echo "	k		size of k-mers [32]."
+	@echo "	w		window size to choose k-mers [32]."
+	@echo "	min_component_size		minimum number of barcodes within a backbone to output [50]."
+	@echo "	minimum_barcode_multiplicity		minimum number of reads a barcode can have [10]."
+	@echo "	maximum_barcode_multiplicity		maximum number of reads a barcode can have [5000]."
+	@echo "	minimizer_multiplicity		maximum number of edges an edges can contribute to [100]."
+	@echo "                             Alternatively, you can specify x, which uses the Q3 + IQR of the minimizer multiplicity distribution"
```

Comment:
Hmm I don't really understand this -- do you specify a parameter 'x'? Or do you specify 'x' for the above parameter?

Reply:
No I meant, you can specify x to that parameter. I've changed the format of that line. Hopefully that clarifies the intention